site/gatsby-site/src/tailwind.css

```diff
@@ -125,6 +120,11 @@
         @apply text-white bg-transparent relative order-1 ml-auto opacity-0 w-0 h-0 overflow-hidden focus:opacity-100 focus:py-[0ch] focus:px-[1ch] focus:w-[unset] focus:h-[unset]  767px:text-xs;
     }
 
+    ul, ol {
+        list-style: revert;
+        padding-left: 2em;
```

Comment:
I mentioned this in another pr, but we can't override global styles like this, it kind of defeats the purpose of using Tailwind. It already has a CSS reset https://tailwindcss.com/docs/preflight.

Reply:
The issue is that tailwind has a `list-style: none` set in their default base settings. So for markdowns in the `mdx` files, it will never show the right list style.
If we choose to use tailwind's `list-style: none` we accept the fact that all markdown list will have no style (no bullets or numbers). 

For example:
<img width="503" alt="image" src="https://user-images.githubusercontent.com/4105289/203116105-82c88726-3b5a-46d8-9961-fee8247c3014.png">
is defined with markdown as with a number list but they will not be displayed as such. My latest commit has that change, let me know what you think.

Reply:
Yeah, I think we should use the plugin typography there; it feels like we are reinventing the wheel.

Reply:
Done! I included the typography plugin for blog posts now. Let me know what you think

Reply:
Nice, seems to be an improvement 🥳